### ISSUE TITLE
[Update] Approve "Missing Scene Performers"

### ISSUE THREAD

**Issue by AdultSun** [stashapp/StashDB-Docs#22]
Needs official approval:
https://guidelines.stashdb.org/docs/scenes/scene-performers/#missing-scene-performers

This one gets pushback from newer contributors, but it doesn't actually say, "Downvote any scenes missing listed male performers." It says they "should be included" if they're listed and that finding missing names from external sources (IAFD, etc.) is "greatly encouraged, but often not required." I don't expect the core of the language to be modified here but there will likely be suggestions to strengthen or weaken the guideline's recommendations on how to enforce it.

**Comment by DogmaDragon** [comment]
It covers the spirit of the project. Do as much as you feel like, but not less than community scrapers provide.

**Comment by Flashy78** [comment]
When I first started contributing, I got quite a few no votes from a particular voter for not having the single name euro guys included, so I can understand how that can really turn off new contributors.
That said, I've only seen a handful of voters do that since. These folks only show up from time to time, and generally if the rest of the submission is good, others will vote yes and get it in.
So I would agree that it's pretty good as is.

**Comment by PhuriousGeorge** [comment]
I agree with all sentiments above - knowing that new contributors may or may not read the guidelines prior to submitting, not sure how much editing would help.  I would suggest at least a poke to read the scene description for additional performer names, as ~20-30% of the time they're listed there

**Comment by Valmox** [comment]
My approach is [generally] to only vote no if they were explicitly listed at the studio source; never because they hadn't searched alternative sources. That said, I've transitioned to only voting no for incorrect info now, with a view to adding them myself if I want to.

We need to consider that Stash lets users exclude male performers on scrape; which means they have to take extra steps at draft submission stage if we want to enforce inclusion.

My preference would be to say that a scene must contain at least one of the performers at the point of submission. The reason I say this is that performers play a role in my voting routine - I've identified a number of dupe scenes (different titles) from checking the performer pages.
